left
image_width: 464
image_height: 400
camera_name: /head/front/cam/image_rect/left
camera_matrix: [237.40791, 0. , 233.43722,           0.     , 239.46768, 206.8979 ,           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients: [0.035341, -0.002923, 0.008238, 0.001785, 0.000000]
rectification_matrix: [ 0.99733076, -0.01062806, 0.07223845,          0.01074024,  0.99994164, -0.00116459,         -0.07222186,  0.00193733,  0.99738671]
projection_matrix: [271.79098, 0. , 202.77231, 0. ,           0.     , 271.79098, 210.75114,   0.     ,           0.     ,   0.     ,   1.     ,   0.     ]
